lib/index.ts

```diff
@@ -13,23 +13,32 @@ const defaultHeader =
  * This module was automatically generated by \`ts-interface-builder\`
  */
 `;
+const ignoreNode = "IGNORE";
+
+interface ICompilerOptions {
+  ignoreGenerics: boolean;
```

Comment:
I mean that if you use `ignoreGenerics?: boolean; ignoreIndexSignature?: boolean`, then you can call `compile(path, {ignoreGenerics: true})` (i.e. only have to list the options that need to be turned on). Also you can then use `options: ICompilerOptions = {}` in line 27. Just simplifies things slightly.

Reply:
Oh I see. I set the single options optional but kept the default value with all the values in there, otherwise I need to check for each option if it is set or not.